hyperspy/_signals/hologram_image.py

```diff
@@ -0,0 +1,329 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+from hyperspy.signals import Signal2D, BaseSignal, Signal1D
+from collections import OrderedDict
+from hyperspy.misc.holography.reconstruct import reconstruct, find_sideband_position, find_sideband_size
+import logging
+import warnings
+import scipy.constants as constants
+
+_logger = logging.getLogger(__name__)
+
+
+class HologramImage(Signal2D):
+    """Image subclass for holograms acquired via off-axis electron holography."""
+
+    _signal_type = 'hologram'
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.sampling = (self.axes_manager[0].scale, self.axes_manager[1].scale)
+        self.f_sampling = np.divide(1, [a * b for a, b in zip(self.data.shape, self.sampling)])
+
+    def find_sideband_position(self, ap_cb_radius=None, sb='lower', show_progressbar=False):
```

Comment:
What about `estimate_sideband_position` instead to emphasize the fact that this is just an estimation?